bindings/cs/Scripts/Create-and-pack-nuget.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+# This script creates 4 folder inside $packagingFolder
+# 1. libmongocrypt-all. This folder contains all platform-specific C libraries.
+# 2. csharp-v{$bindingsReleaseVersion}. After all, this folder will containts the fully ready source code.
+# 3. downloaded_mongocrypt. This folder contains the used in the libmongocrypt platform-specific C libraries.
+# 4. artifacts. After all, the generated .nupkg files will be placed here.
+#
+# Input data:
+# $libmongocryptAllUrl - this is the link on EG patch that contains all required platform-specific C libraries.
+
+packagingFolder="c:/build"
+bindingsReleaseVersion="1.2.1"
+fork="https://github.com/DmitryLukyanov/libmongocrypt.git"
```

Comment:
This should point to the main repo by default.

Reply:
agreed